google_shopping.js

```diff
@@ -2,4 +2,83 @@ var data = require("./products.json")
 
 // Write your solutions below
 
-console.log(data["items"]);
\ No newline at end of file
+/*
+1.) The `kind` of results you're are dealing  are `shopping#products`.
+Go through the `items` and find all results that have `kind` of `shopping#product`.
+How many are there? Where else is this count information stored in the search results?
+*/
+
+console.log("There are " + data["items"].length + " items in the google shopping list.");
+console.log("~~~~~~~~~~~~~~~~~~~~");
+
```

Comment:
This is one way to see how many items there are in total. To find how many items have kind 'shopping#product' you can perform a loop as follows:

```var numberOfShoppingProducts = null; // Set variable to keep track of products whose type matches "shopping#products"

for (var i = 0; i < data["items"].length; i += 1) {
	if (data["items"][i].kind == 'shopping#product') {
	numberOfShoppingProducts++
	};	
}; // Will check for items that match 'kind' 'shopping#product' and add them to tracking variable```